.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: cpp
 
-dist: trusty
+#dist: trusty
```

Comment:
Just delete this line.  No need to comment it out because the source control system will have the former value in history 

Remove the other commented out lines from the rest of this file